tests/broker-test.py

```diff
@@ -39,7 +39,8 @@
     _AZURE_CLI,
     authority="https://login.microsoftonline.com/organizations",
     enable_broker_on_mac=True,
-    enable_broker_on_windows=True)
+    enable_broker_on_windows=True,
+    enable_broker_on_linux=True)
```

Comment:
**nit:** Write it this way, so that future addition will result in a cleaner PR.
```suggestion
    enable_broker_on_linux=True,
    )
```